app/middlewares/user.js

```diff
@@ -12,3 +13,22 @@ exports.userVerifyUserByEmail = async (req, _, next) => {
     return next(errors.databaseError('Error to verify email'));
   }
 };
+
+exports.confirmPassword = async (req, _, next) => {
+  try {
+    const { body } = req;
+    const { email, password } = body;
+    const user = await userService.findUserByEmail(email);
+    if (!user) return next(errors.badRequestError('User incorrect'));
```

Comment:
For security reasons we cannot say which of the data failed (email or password), for both messages, we should indicate that the email or password failed and thus we add a little security.